extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -107,7 +114,18 @@ public static PhotoModel convertToPhotoModel(
         albumId.orElse(null),
         false  /*inTempStore*/,
         null  /*sha1*/,
-        mediaItem.getUploadedTime());
+        getCreationTime(mediaItem));
+  }
+
+  private static Date getCreationTime(GoogleMediaItem mediaItem) throws ParseException  {
+    // cannot be empty or null based. Verified the backend code.
+    try {
+      return CREATION_TIME_FORMAT.parse(mediaItem.getMediaMetadata().getCreationTime());
+    } catch (ParseException parseException) {
+      throw new ParseException(String.format("Failed to parse the string %s to get creationTime for "
+              + "MediaItem %s .", mediaItem.getId(), mediaItem.getMediaMetadata().getCreationTime()),
+          parseException.getErrorOffset());
```

Comment:
getErrorOffset still not being used

Reply:
It's a parameter being passed and not part of the string format